<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Modelos</class>
 <widget class="QDialog" name="Modelos" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Selecione o Modelo</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/rs.qrc" >:/png/buttons/16/frameprint.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Modelo</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>51</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btAdd" >
       <property name="text" >
        <string>Adicionar</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/rs.qrc" >:/png/buttons/16/edit_add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btEdit" >
       <property name="text" >
        <string>Alterar</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/rs.qrc" >:/png/buttons/16/cell_layout.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btDelete" >
       <property name="text" >
        <string>Excluir</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/rs.qrc" >:/png/buttons/16/button_cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QTextBrowser" name="textBrowser" />
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/rs.qrc" >:/png/buttons/16/agt_action_success.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancelar</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/rs.qrc" >:/png/buttons/16/agt_action_fail.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../res/rs.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Modelos</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>478</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Modelos</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>578</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
